src/core/core.layoutService.js

```diff
@@ -10,24 +10,58 @@ module.exports = function(Chart) {
 	Chart.layoutService = {
 		defaults: {},
 
-		// Register a box to a chartInstance. A box is simply a reference to an object that requires layout. eg. Scales, Legend, Plugins.
+		/**
+		 * Register a box to a chartInstance.
+		 * A box is simply a reference to an object that requires layout. eg. Scales, Legend, Title.
+		 * @method Chart.layoutService.addBox
+		 * @param {Core.Controller} chartInstance the chart to use
+		 * @param {Object} box the box to add
+		 * @param {Number} box.order the order of the boxes. Higher numbers are further away from the chart area
+		 * @param box.options Options for configuring the box
```

Comment:
Why are there some options under `box.options` and some others directly under `box`?